test-result-summary-client/src/Dashboard/Widgets/Graph/Renaissance.jsx

```diff
@@ -135,19 +135,19 @@ export default class Renaissance extends Component {
 
 				test.testData.metrics.forEach(( metric, i)=> {
 					if ( metric.name === "akka-uct" ) {
-						akkaUct = metric.value[0];
+						akkaUct = calculateMean(metric.value);
```

Comment:
Could we use `math.mean()`?

Reply:
I changed those to `math.mean()` and it would check if `metric.value` is not empty before calculate the mean.